src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/commands.py

```diff
@@ -37,44 +37,34 @@ def load_command_table(self, _):
         g.custom_command('backup-properties set', 'set_backup_properties', client_factory=backup_storage_configs_cf)
         g.custom_command('delete', 'delete_vault', confirmation=True)
 
-    with self.command_group('backup container', backup_custom, client_factory=backup_protection_containers_cf) as g:
+    with self.command_group('backup container', backup_custom_base, client_factory=backup_protection_containers_cf) as g:
         g.show_command('show', 'show_container')
-        g.command('list', 'list_containers', table_transformer=transform_container_list)
-
-    with self.command_group('backup azure-wl container', backup_custom, client_factory=backup_protection_containers_cf) as g:
-        g.show_command('show', 'show_wl_container')
-        g.command('list', 'list_wl_containers', table_transformer=transform_wl_container_list)
-        g.command('register', 'wl_register_container')
+        g.command('list', 'list_containers', table_transformer=transform_containers_list)
+        g.command('register', 'register_wl_container', client_factory=protection_containers_cf)
+        g.command('re-register', 're_register_wl_container', client_factory=protection_containers_cf, confirmation=True)
+        g.command('unregister', 'unregister_wl_container', client_factory=protection_containers_cf, confirmation=True)
 
-    with self.command_group('backup policy', backup_custom, client_factory=protection_policies_cf) as g:
+    with self.command_group('backup policy', backup_custom_base, client_factory=protection_policies_cf) as g:
         g.command('get-default-for-vm', 'get_default_policy_for_vm')
-        g.show_command('show', 'show_policy')
-        g.command('list', 'list_policies', client_factory=backup_policies_cf, table_transformer=transform_policy_list)
+        g.show_command('show', 'show_policy', table_transformer=transform_wl_policy_show)
+        g.command('list', 'list_policies', client_factory=backup_policies_cf, table_transformer=transform_policies_list)
```

Comment:
any specific reason to rename the function?

Reply:
This function splits flow for ISVM and Workload formatter